src/Checkbox/web/Checkbox.js

```diff
@@ -8,6 +9,10 @@ import Input from './Input';
 import Box from './Box';
 import Tick from './Tick';
 
+const CheckboxWrapper = styled.div`
```

Comment:
We should not create `CheckboxWrapper` or any such wrappers for a specific case. We already have enhancer components `Position, Size, Space, Flex` to cater all these requirements.